docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+version: "3.5"
+
+services:
+
+  php:
+    build: docker/php
+    hostname: 'projectaanvraag.local'
+    container_name: projectaanvraag
+    volumes:
+      - .:/var/www/html:cached
+    working_dir: /var/www/html
+    ports:
+      - '8500:80'
```

Comment:
Maybe we just take port `8000` like for `udb3-backend`?

I guess port `8500` was used to avoid conflicts, but Redis and MySQL reuse the same ports, so we have other conflicts. Therefore I would go for consistency.

Reply:
Indeed, I've changed it.